platform/security/sts/security-sts-ldapclaimshandler/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -70,7 +70,7 @@
 
 	</OCD>
 
-	<Designate factoryPid="Claims_Handler_Manager">
+	<Designate factoryPid="Claims_Handler_Manager" pid="Claims_Handler_Manager">
 	    <Object ocdref="Claims_Handler_Manager"/>
```

Comment:
I don't think the pid is necessary? The factoryPid is the one referenced from the blueprint.